modules/integration/test-ui-sp-editor/src/test/java/org/wso2/sp/selenium/spTestCases/testCases.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.sp.selenium.spTestCases;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.wso2.sp.selenium.components.pageObjects.headerContainerObjects.*;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class testCases extends baseTestClass {
```

Comment:
Class name should start with Capital letter.

Reply:
Fixed in commit 3f8b09e10c14BexZMoP1gqvSbLZSfYigjUvfcXkroScK